tests/testerina-integration-test/src/test/java/org/ballerinalang/testerina/test/BaseTestCase.java

```diff
@@ -78,6 +79,11 @@ public void initialize() throws BallerinaTestException, IOException {
                 Paths.get("src", "test", "resources", "project-based-tests/test-report-tests").toAbsolutePath();
         reportTestProjectPath = tempProjectDirectory.resolve("test-report");
         FileUtils.copyFolder(originalReportTestProj, reportTestProjectPath);
+
+        Path outsideTestsProj =
+                Paths.get("src", "test", "resources", "project-based-tests/outside-tests").toAbsolutePath();
```

Comment:
Won't this break in Windows? Why don't we have something like this?
```suggestion
                Paths.get("src", "test", "resources", "project-based-tests", "outside-tests").toAbsolutePath();
```

Reply:
Thanks for the pointer @parkavi11 
You are correct. We should either use "File.separator" or use the approach you suggested. I'll correct it,